src/templates/base_register.html

```diff
@@ -79,15 +78,14 @@
     })
 });
 </script>
-
-{% endblock %}
-{% block extrajs %}
```

Comment:
Because " . js script " are wriiten under the " {% block extrahead %}.....{%endblock%} " and after that .js files are written under {% block js%}.....{%endblock%}" and It was fine.  But after this, there was again comments for " extrajs " without any reason. So , I delete it.